static/scripts/rewards/app-state.ts

```diff
@@ -1,5 +1,5 @@
 import { JsonRpcProvider } from "@ethersproject/providers";
-import { networkExplorers } from "./constants";
+import { networkExplorers } from "@keyrxng/rpc-handler/dist/esm/src/constants";
```

Comment:
Seems like this can be improved.

Reply:
you mean the import path? Intellisense works fine with it but it is pretty verbose

Reply:
Normally the packages barrel the useful objects up to the package root so that it can be imported like `@ubiquity/rpc-handler`